tests/init.lua

```diff
@@ -1,35 +0,0 @@
--- From https://github.com/folke/noice.nvim/blob/main/tests/init.lua
-
-local M = {}
-
-function M.root(root)
-  local f = debug.getinfo(1, "S").source:sub(2)
-  return vim.fn.fnamemodify(f, ":p:h:h") .. "/" .. (root or "")
-end
-
----@param plugin string
-function M.load(plugin)
-  local name = plugin:match(".*/(.*)")
-  local package_root = M.root(".tests/site/pack/deps/start/")
```

Comment:
I'll check where this is getting cloned to avoid populating usable paths here like ~/.local/share/nvim/site/pack/vendor/start/